src/sage/schemes/hyperelliptic_curves_smooth_model/hyperelliptic_constructor.py

```diff
@@ -0,0 +1,279 @@
+r"""
```

Comment:
Maybe just name this file ``constructor.py`` for consistency with the existing elliptic and hyperelliptic packages?